bulk.go

```diff
@@ -259,7 +255,7 @@ func (a *Account) bulkDeleteSingle(objects []*Object, containers []*Container, o
 // Implementation of BulkDelete() for servers that *do* support bulk deletion.
 // This function is called *after* chunking, so `len(names) <=
 // account.Capabilities.BulkDelete.MaximumDeletesPerRequest`.
-func (a *Account) bulkDelete(names []string, opts *RequestOptions) (int, int, error) {
+func (a *Account) bulkDelete(names []string, opts *RequestOptions) (NumberDeleted, NumberNotFound int, err error) {
```

Comment:
Local variable names should not be upper-case.

```suggestion
func (a *Account) bulkDelete(names []string, opts *RequestOptions) (numDeleted, numNotFound int, err error) {
```